admin-client/src/components/LabeledItem.svelte

```diff
@@ -0,0 +1,13 @@
+<script>
+  export let label = 'label';
+  export let value = null;
```

Comment:
does this value get redefined?

Reply:
Ya, that's the svelte way of creating a default value for a property.  Any variable in a component with the `export` syntax is a value you can pass to a component and the passed value will take precedent in the prop. 

ie
```svelte
// passing a value 
<LabeledItem label={'Hello'} value={'World'} />

// Shorthand version
<script>
  const label = 'Hello';
  const value = 'World';
</script>

<LabelItem {label} {value} />
```